### PR TITLE
stable-2.0 | Backport fixes for ARM & PPC

### PR DESCRIPTION
**Pull request by fidencio**
This contains two backports:
**ci: updates to kata setup scripts for ppc64le**

    - kubenetes package installation for ppc64le
    - virtiofsd installation

    Fixes: #3185

Signed-off-by: Jing Wang <jing.wang4@ibm.com>
(cherry picked from commit 0febfe00f8c8bbfc45ce27ec852320a698aef1ad)

**ci: workaround to fix arm ci failure.**

Currently, before containerd integration test, containerd service is
stopped., which will disable docker service. Then, the TestImageLoad
will fail for it depends on docker service.

Here, we restart docker before containerd test to avoid that failure.

But these tests have been working well for years. I can't tell why it
is.

Also do cleanup before k8s initialization to avoid lots of bizarre
issues.
for example:
1. [ERROR Port-10250]: Port 10250 is in use
2. Unfortunately, an error has occurred:
                timed out waiting for the condition

Fixes: #3478
Signed-off-by: Jianyong Wu <jianyong.wu@arm.com>
(cherry picked from commit d7ca6409275b88ef12fad30ec7c8c188e8766f47)

### CODE REVIEW

**Comment by fidencio** [comment]
@GabyCT, @devimc, @wainersm, if you happen to have an "approval" to spare ...

**Review by GabyCT — APPROVED** [review]

**Comment by GabyCT** [comment]
@devimc the `vfio` CI is failing, is something that is missing to do a backport? or it is a known failure

**Comment by devimc** [comment]
@GabyCT I think this PR needs a rebase... :thinking:

**Review by devimc — APPROVED** [review]

**Comment by GabyCT** [comment]
merging this PR as vfio is fixed and @devimc approved it

**Comment by fidencio** [comment]
It was opened before the vfio work was merged.  Knowing the VFIO fix already landed, shall we rebase and re-run the whole CI or can we go ahead with this as it is?

**Comment by fidencio** [comment]
> It was opened before the vfio work was merged. Knowing the VFIO fix already landed, shall we rebase and re-run the whole CI or can we go ahead with this as it is?

Solved :-)

**Pull request merged**
